snapshots/types/util.go

```diff
@@ -4,12 +4,12 @@ import (
 	protoio "github.com/gogo/protobuf/io"
 )
 
-// WriteExtensionItem writes an item payload for current extension snapshotter.
-func WriteExtensionItem(protoWriter protoio.Writer, item []byte) error {
+// WriteExtensionPayload writes an extension payload for current extension snapshotter.
+func WriteExtensionPayload(protoWriter protoio.Writer, payload []byte) error {
```

Comment:
Can we mention this API change in the changelog please?

Reply:
sure, done